app/src/main/java/org/gdg/frisbee/android/api/PlusApi.java

```diff
@@ -0,0 +1,18 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.BuildConfig;
+import org.gdg.frisbee.android.api.model.plus.ImageInfo;
+import org.gdg.frisbee.android.api.model.plus.Person;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+
+public interface PlusApi {
+    @GET("people/{gplusId}?fields=image&key=" + BuildConfig.IP_SIMPLE_API_ACCESS_KEY)
+    Call<ImageInfo> getImageInfo(@Path("gplusId") String gplusId);
+
+    @GET("people/{gplusId}?fields=image,aboutMe,tagline,urls,url,cover,displayName&key="
```

Comment:
Now I realized this duplication, this key can be added through OkHttp interceptors.